imagecraft/services/lifecycle.py

```diff
@@ -19,14 +19,10 @@
 from typing import cast
 
 from craft_application import LifecycleService
-from craft_parts import Features
 from overrides import override  # type: ignore[reportUnknownVariableType]
 
 from imagecraft.models.project import Project
 
-# Enable the craft-parts features that we use
-Features(enable_overlay=True)
```

Comment:
where is this being enabled now? e.g. https://github.com/canonical/rockcraft/blob/main/rockcraft/application.py#L82

Reply:
Nowhere :). I will enable this (or most probably the partition feature first) as a following step when implementing following specs (the original one is not mentioning it so I want to keep it clean for now).